libc/src/sys/epoll/linux/CMakeLists.txt

```diff
@@ -3,39 +3,60 @@ add_entrypoint_object(
   SRCS
     epoll_wait.cpp
   HDRS
+    # TODO: these 2 should not be necessary
+    ../../../signal/sigset_t.h
+    ../../../time/struct_timespec.h
+
     ../epoll_wait.h
   DEPENDS
-    libc.include.sys_epoll
     libc.include.sys_syscall
     libc.src.__support.OSUtil.osutil
     libc.src.errno.errno
+    libc.src.sys.epoll.struct_epoll_event
+    # TODO: why don't these work for overlay mode?
```

Comment:
does setting `-DLIBC_CMAKE_VERBOSE_LOGGING=ON` in your cmake help?

Reply:
obsolete as of https://github.com/llvm/llvm-project/pull/87017/commits/4e2833ab58e1d64c95cda19338370899f62d15fd